packages/xo-web/src/xo-app/settings/remotes/remote.js

```diff
@@ -360,7 +370,7 @@ export default decorate([
                 </span>
                 <Toggle
                   className='align-middle pull-right'
-                  onChange={effects.setInsecure}
+                  onChange={effects.setUseVhdDirectory}
```

Comment:
Since `useVhdDirectory` must always be `true` for `S3` remote, it would be nice to disable the toggle no ?